docs/standard/garbage-collection/fundamentals.md

```diff
@@ -74,11 +74,11 @@ manager: "wpickett"
   
     -   Committed. The block of memory is assigned to physical storage.  
   
--   Virtual address space can get fragmented. This means that there are free blocks, also known as holes, in the address space. When a virtual memory allocation is requested, the virtual memory manager has to find a single free block that is large enough to satisfy that allocation request. Even if you have 2 GB of free space, the allocation that requires 2 GB will be unsuccessful unless all of that space is in a single address block.  
+-   Virtual address space can get fragmented. This means that there are free blocks, also known as holes, in the address space. When a virtual memory allocation is requested, the virtual memory manager has to find a single free block that is large enough to satisfy that allocation request. Even if you have 2 GB of free space, the allocation that requires 2 GB will be unsuccessful unless all of that free space is in a single address block.  
   
 -   You can run out of memory if you run out of virtual address space to reserve or physical space to commit.  
   
- Your page file is used even if physical memory pressure (that is, demand for physical memory) is low. The first time your physical memory pressure is high, the operating system must make room in physical memory to store data, and it backs up some of the data that is in physical memory to the page file. That data is not paged until it is needed, so it is possible to encounter paging in situations where the physical memory pressure is very low.  
+ Your page file is used even if it will demand more physical memory then increase memory's pressure. The first time your physical memory pressure is high, the operating system must make room in physical memory to store data, and it backs up some of the data that is in physical memory to the page file. That data is not paged until it is needed, so it is possible to encounter paging in situations where the physical memory pressure is very low.  
```

Comment:
> Your page file is used even if it will demand more physical memory then increase memory's pressure.

I don't understand what is this sentence trying to say. I think the original phrasing is better.